tensorflow/tools/docker/Dockerfile.devel-cpu-mkl

```diff
@@ -0,0 +1,127 @@
+FROM ubuntu
+
+LABEL maintainer="Clayne Robison<clayne.b.robison@intel.com>"
+
+# It is possible to override these for releases.
+ARG TF_BRANCH=r1.4
+ARG BAZEL_VERSION=0.7.0
+ARG WHL_DIR=/whl
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        build-essential \
+        curl \
+        git \
+        golang \
+        libcurl3-dev \
+        libfreetype6-dev \
+        libzmq3-dev \
+        pkg-config \
+        python-dev \
+        python-pip \
+        rsync \
+        software-properties-common \
+        unzip \
+        zip \
+        zlib1g-dev \
+        openjdk-8-jdk \
+        openjdk-8-jre-headless \
+        wget \
+	locate \
+	vim \
```

Comment:
we need vim?

Reply:
Yes. The purpose of this container is to enable engineers to get a common build with common components to work at the command line. No intention of using Jupyter and web interface.

Reply:
Well, usually this isn't done this way. If someone builds app they'll write their own Dockerfile, FROM this one and use COPY or ADD to put code inside. Editing anything inside container is considered antipattern as image is not easy to reproduce.

Reply:
True.